src/ApparentHorizons/StrahlkorperGr.cpp

```diff
@@ -193,14 +193,15 @@ Scalar<DataVector> spin_function(
     const tnsr::I<DataVector, 3, Frame>& unit_normal_vector,
     const Scalar<DataVector>& area_element,
     const tnsr::ii<DataVector, 3, Frame>& extrinsic_curvature) noexcept {
-  auto sin_theta = make_with_value<Scalar<DataVector>>(area_element, 0.0);
+  DataVector temp(get(area_element).size(), 0.0);
 
   auto extrinsic_curvature_theta_normal_sin_theta =
       make_with_value<Scalar<DataVector>>(area_element, 0.0);
   auto extrinsic_curvature_phi_normal =
       make_with_value<Scalar<DataVector>>(area_element, 0.0);
   for (size_t i = 0; i < 3; ++i) {
-    DataVector extrinsic_curvature_dot_normal(get(sin_theta).size(), 0.0);
+    DataVector& extrinsic_curvature_dot_normal = temp;
```

Comment:
`Scalar<DataVector>& extrinsic_curvature_dot_normal = temp;

Reply:
I'd still move that outside of the for loop :)